docs/docusaurus.config.ts

```diff
@@ -67,6 +67,16 @@ export default {
           label: 'GitHub',
           position: 'right',
         },
+        {
+          href: 'https://noir-lang.github.io/noir/docs/acir/circuit/index.html',
+          label: 'ACIR reference',
+          position: 'right',
+        },
+        {
+          href: 'https://noir-lang.github.io/noir/docs/nargo/index.html',
+          label: 'Source reference',
+          position: 'right',
+        },
```

Comment:
Did people ask for Nargo references?

Reply:
they did not, but thought i'd add in the interest of visibility. I will remove it for simplicity/focus